app/assets/stylesheets/forever_style_guide/modules/_pnav.scss

```diff
@@ -0,0 +1,464 @@
+  height: 22px;
+  max-width: 138px;
+  @include transition_fast(height, max-width);
+}
+
+.f-pnav__ecom-list {
+  display: flex;
+  flex: 1 0 auto;
+  justify-content: center;
+  align-items: stretch;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.f-pnav__ecom-item {
+  /* we want to ul to remain and grow, so just hide the items */
+  display: none;
+}
+
+.f-pnav__ecom-link {
+  position: relative;
+  display: flex;
+  align-items: center;
+  padding-right: 4px;
+  padding-left: 4px;
+  font-family: $font-face-semibold;
+  font-size: $font-size-200;
+
+  &.f-pnav__ecom-link-deals {
+    @include deals_link_base();
+  }
+
+  &::before {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    height: 4px;
+    width: 100%;
+    background-color: $color-forever_blue;
+    content: '';
+    transform: scaleX(0);
+    @include transition_fast(transform);
+  }
+
+  &.is-active,
+  &:hover {
+    &::before {
+      transform: scaleX(1);
+    }
+  }
+}
+
+.f-pnav__icon-list {
+  position: relative;
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.f-pnav__icon-item {
+  padding: 0 6px;
+  font-size: 21px;
+
+  /* needed for notification dropdown icon */
+  .fa:hover {
+    @include transition_normal(color);
+    color: $color-forever_blue;
+  }
+
+}
+
+.f-pnav__icon-link-cart {
+  position: relative;
+
+  .f-pnav__icon-link-cart-count {
+    position: absolute;
+    top: 0;
+    right: -10px;
+    width: 18px;
+    height: 18px;
+    border-radius: 4px;
+    background-color: #e22626;
+    color: $color-white;
+    font-size: $font-size-000;
+    font-family: $font-face-semibold;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    pointer-events: none;
+  }
+
+}
+
+.f-pnav__user-list {
+  display: flex;
+  align-items: center;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+
+  .f-pnav__user-link {
+    font-family: $font-face-semibold;
+    text-transform: uppercase;
+
+    &.btn {
+      padding: 7px;
+      font-size: 12px;
+      height: 30px;
+    }
+  }
+
+  .f-pnav__user-link-signup {
+    &.btn {
+      color: $color-white;
+    }
+  }
+
+  .f-pnav__user-link-signin {
+    &.btn {
+      color: $color-gray-600;
+
+      &:hover {
+        text-decoration: none;
+      }
+
+    }
+  }
+}
+
+.f-pnav__user-item-avatar {
+  padding-left: 10px;
+
+  /* use below instead of display to hide so we retain padding */
+  visibility: hidden;
+  width: 0;
+
+  .avatar {
+    padding: 0;
+    margin-right: 0;
+  }
+}
+
+.f-pnav__user-item-logged-out {
+  display: none;
+}
+
+.f-pnav__user-item-side-toggle {
+  cursor: pointer;
+  padding: 10px 10px 10px 5px;
+  margin-right: -10px;
+
+  .f-pnav__user-icon-bar {
+    background-color: $color-gray-600;
+    width: 20px;
+    display: block;
+    height: 2px;
+    border-radius: 1px;
+    @include transition_fast(background-color, margin);
+  }
+
+  .f-pnav__user-icon-bar + .f-pnav__user-icon-bar {
```

Comment:
is there a less brittle way to target this element?

Reply:
I was following what twbs had as a basis
![image](https://user-images.githubusercontent.com/2379769/54556843-28321580-4990-11e9-8157-3e98d42874b4.png)

Aside from giving them explicit classes or targeting based on nth child (also brittle).

Reply:
ahh so this is just saying :not:first-child? That might be as brittle, but clearer code?

Reply:
Sort of the same, certainly reads better. Change made.